transport/tchannel/handler.go

```diff
@@ -142,7 +140,7 @@ func (h handler) callHandler(ctx context.Context, call inboundCall) error {
 		return err
 	}
 
-	return h.Handler.Handle(ctx, tchannelOptions, treq, rw)
+	return h.Handler.Handle(ctx, transportOptions, treq, rw)
```

Comment:
Sort-of. Transport options is where transports specify additional
requirements. Right now, TChannel is the only transport that has non-standard
requirements, and they're constant: "Don't envelope Thrift payloads."

If in the future we add more options for TChannel, `transportOptions` may
become,

    transport.Options{}.Merge(
        thrift.DisableEnvelopingForTransport,
        other.OptionA(..),
        OptionB(..),
    )